corehq/apps/export/models/new.py

```diff
@@ -251,9 +249,6 @@ class ExportInstance(Document):
     # Whether to split multiselects into multiple columns
     split_multiselects = BooleanProperty(default=False)
 
-    # Whether to automatically convert dates to excel dates
-    transform_dates = BooleanProperty(default=False)
```

Comment:
ok good to get your opinion, my thought was essentially we're special casing for transform dates when it could just be another transform. it'd be kind of nice to be able to look at an `ExportColumn` and know exactly what changes will be made to the value. though an argument can be made here that this is worthy of being a global boolean since it always will affect all columns. here was the commit where the idea was born: https://github.com/dimagi/commcare-hq/pull/10236#commitcomment-15953899